src/core/ports/imageport.cpp

```diff
@@ -97,7 +97,7 @@ void ImageOutport::clear() {
 bool ImageOutport::hasEditableData() const { return static_cast<bool>(image_); }
 
 size2_t ImageOutport::getLargestReqDim() const {
-    if (requestedDimensions_.empty()) return size2_t(0, 0);
+    if (requestedDimensions_.empty()) return size2_t(1, 1);
```

Comment:
Why not 0?

Reply:
Im trying to be a bit defensive and avoid getting invalid FBOs, 0,0 is not a popular size for a fbo it seems